Landing-page.css

```diff
@@ -0,0 +1,842 @@
+@import url('https://fonts.googleapis.com/css2?family=Nunito:wght@300;400;600;800&display=swap');
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    background-color: #fff;
+    font-family: Nunito;
+    overflow-x: none;
+}
+
+.container {
+    width: 90vw;
+    margin: auto;
+
+}
+
+ul {
+    list-style-type: none;
+}
+
+/**
+** HEADER
+**/
+
+header {
+    display: flex;
+    padding-top: 2.6rem; 
+}
+
+header .header-lt {
+    
+}
+
+header .header-lt h2 {
+    color: #c4c4c4;
+}
+
+header .header-rt {
+    /*
+    display: flex;
+    justify-content: space-between;
+    width: 200px;
+    */
+    display: flex;
+    justify-content: space-around;
+    margin-left: auto;
+}
+
+.search-box {
+    display: flex;
+    align-items: center;
+    border-radius: 2rem;
+}
+
+.search-box svg {
+    height: 2.3rem;
+    width: 2.3rem;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/codeclannigeria/CodeClanUI/pullRequest?prid=5575014)